packages/hint/src/lib/utils/dom/create-new-async-document.ts

```diff
@@ -0,0 +1,13 @@
+import { IAsyncHTMLDocument } from '../../types/async-html';
```

Comment:
We used `async` because that's how CDP works but with this probably we don't need to make the API async. It's probably a bit more changes but maybe we should make it `sync` again and update the names?

Reply:
sounds good.